router/router.go

```diff
@@ -631,21 +572,14 @@ func (worker *workerT) processDestinationJobs() {
 				//			router_delivery_exceeded_timeout -> goes to zero
 				ch := worker.trackStuckDelivery()
 
-				resultSetID := destinationJob.JobMetadataArray[0].ResultSetID
-
-				if worker.localResultSet.id < resultSetID {
-					resultSet := worker.rt.getResultSet(resultSetID)
-					worker.localResultSet.id = resultSetID
-					worker.localResultSet.resultSetBeginTime = time.Now()
-					worker.localResultSet.timeAllotted = resultSet.timeAllotted
-				}
-
 				// Assuming twice the overhead - defensive: 30% was just fine though
 				// In fact, the timeout should be more than the maximum latency allowed by these workers.
 				// Assuming 10s maximum latency
-				elapsed := time.Since(worker.localResultSet.resultSetBeginTime)
-				threshold := time.Duration(2.0 * math.Max(float64(worker.localResultSet.timeAllotted), float64(10*time.Second)))
+				elapsed := time.Since(destinationJob.JobMetadataArray[0].LatestResultSetTime)
+				threshold := worker.rt.routerTimeout
 				if elapsed > threshold {
+					fmt.Println("TimeStamp", destinationJob.JobMetadataArray[0].LatestResultSetTime)
```

Comment:
```suggestion
					fmt.Println("TimeStamp", worker.latestAssignedTime)
```

Reply:
Removed these.. added them during debugging but forgot to clean up